lib/internal/event_target.js

```diff
@@ -10,6 +10,8 @@ const {
   ObjectDefineProperty,
   Symbol,
   SymbolFor,
+  SafeWeakSet,
+  ObjectGetOwnPropertyDescriptor,
```

Comment:
This should be sorted alphabetically:
```suggestion
  ObjectGetOwnPropertyDescriptor,
  SafeWeakSet,
  Symbol,
  SymbolFor,
```